CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ and this project adheres to
 - Add graylog service
 - Add edx lms service
 - Add keycloak service
+- Add edX course generator
```

Comment:
It seems there is whitespace since the bullet point of "Add graylog service", isn't it?